prov/tcp/src/tcpx.h

```diff
@@ -120,6 +127,7 @@ struct tcpx_pep {
 	struct fi_info		*info;
 	SOCKET			sock;
 	struct tcpx_cm_context	cm_ctx;
+	struct tcpx_port_range	port_range;
```

Comment:
We don't need to store this per passive endpoint.  A single global for the module is sufficient.